CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+#[[
+This source file is part of the Swift System open source project
+
+Copyright (c) 2020 Apple Inc. and the Swift System project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+cmake_minimum_required(VERSION 3.19.0)
+
+if(POLICY CMP0077)
+  cmake_policy(SET CMP0077 NEW)
+endif()
+
+if(POLICY CMP0091)
+  cmake_policy(SET CMP0091 NEW)
+endif()
+
+project(SwiftMarkdown
+  LANGUAGES C Swift)
+
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules)
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+
+set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
+set(CMAKE_Swift_COMPILE_OPTIONS_MSVC_RUNTIME_LIBRARY MultiThreadedDLL)
+
+include(FetchContent)
+include(SwiftSupport)
+
+set(_SM_VENDOR_DEPENDENCIES)
+
+set(BUILD_EXAMPLES NO)
+set(BUILD_TESTING NO)
+
+find_package(ArgumentParser CONFIG)
```

Comment:
Where are the ArgumentParser and cmark-gfm targets aliased?

Reply:
They are here.

- https://github.com/apple/swift-cmark/pull/69
- https://github.com/apple/swift-argument-parser/blob/main/CMakeLists.txt

Could you please add the PR's dependency to the body here? @compnerd 

(See the PR template here: https://github.com/apple/swift-markdown/blob/main/.github/pull_request_template.md)

Reply:
To be clear, I know where the repositories are - this comment is about `find_package` namespacing its target names, where as `FetchContent` will not. See eg. https://github.com/apple/swift-format/pull/677/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR76 for what I mean.

Reply:
Ugh, I think I missed that part of the change, I'll fix that up.

Reply:
Ah, swift-argument-parser and cmark both do not namespace their export targets, so there is no need for the aliasing.

Reply:
@Kyle-Ye - there are no dependencies for this change, it is fully standalone, it just adds the CMake build rules.